.github/workflows/full-android.yaml

```diff
@@ -64,11 +64,8 @@ jobs:
               # We do not need pigweed cross compile here because we use android NDK
               # compilers, so removing rust and arm compilers saves 2.5GB
```

Comment:
maybe slight update in comments because arm only saves 1.5GB. Unsure if we can restate so that we do not need to keep track of actual  size.

Reply:
:facepalm: I didn't see this until now. I went with `Removing this package save significant amount of space.`